apps/condo/domains/notification/templates.js

```diff
@@ -1,135 +1,150 @@
     RESET_PASSWORD_MESSAGE_TYPE,
     SMS_VERIFY_CODE_MESSAGE_TYPE,
     SHARE_TICKET_MESSAGE_TYPE,
-    EMAIL_TRANSPORT,
-    SMS_TRANSPORT,
     DEVELOPER_IMPORTANT_NOTE_TYPE,
     CUSTOMER_IMPORTANT_NOTE_TYPE,
-    MESSAGE_FORWARDED_TO_SUPPORT,
+    MESSAGE_FORWARDED_TO_SUPPORT_TYPE,
+    TICKET_ASSIGNEE_CONNECTED_TYPE,
+    TICKET_EXECUTOR_CONNECTED_TYPE,
+    TICKET_STATUS_CHANGED_TYPE,
 } = require('./constants/constants')
 
-async function renderTemplate (transport, message) {
-    if (!MESSAGE_TRANSPORTS.includes(transport)) throw new Error('unexpected transport argument')
+const SERVER_URL = conf.SERVER_URL
 
-    // TODO(pahaz): we need to decide where to store templates! HArDCODE!
-    // TODO(pahaz): write the logic here!
-    //  1) we should find message template by TYPE + LANG
-    //  2) we should render the template and return transport context
+const getInviteNewEmployeeMessage = (message) => {
+    const { organizationName, inviteCode } = message.meta
 
-    const serverUrl = conf.SERVER_URL
+    if (message.lang === EN_LOCALE) {
+        return {
+            subject: 'You are invited to join organization as employee',
+            text: `Organization "${organizationName}" invited you as employee.\n` +
+                `Click to the link to join: ${SERVER_URL}/auth/invite/${inviteCode}`,
+        }
+    }
 
-    if (message.type === INVITE_NEW_EMPLOYEE_MESSAGE_TYPE) {
-        const { organizationName, inviteCode } = message.meta
+    if (message.lang === RU_LOCALE) {
+        return {
+            subject: 'Вас пригласили присоединиться к организации в качестве сотрудника',
+            text: `Администратор организации "${organizationName}" приглашает вас в качестве сотрудника.\n` +
+                `Перейдите по ссылке, чтобы присоединиться: ${SERVER_URL}/auth/invite/${inviteCode}`,
+        }
+    }
+}
 
+const getDirtyInviteNewEmployeeMessage = (message, transport) => {
+    const { organizationName } = message.meta
+    if (transport === EMAIL_TRANSPORT) {
         if (message.lang === EN_LOCALE) {
             return {
                 subject: 'You are invited to join organization as employee',
                 text: `Organization "${organizationName}" invited you as employee.\n` +
-                    `Click to the link to join: ${serverUrl}/auth/invite/${inviteCode}`,
+                    `Click to the link to join: ${SERVER_URL}/auth/signin`,
             }
-        } else if (message.lang === RU_LOCALE) {
+        }
+
+        if (message.lang === RU_LOCALE) {
             return {
                 subject: 'Вас пригласили присоединиться к организации в качестве сотрудника',
                 text: `Администратор организации "${organizationName}" приглашает вас в качестве сотрудника.\n` +
-                    `Перейдите по ссылке, чтобы присоединиться: ${serverUrl}/auth/invite/${inviteCode}`,
+                    `Перейдите по ссылке, чтобы присоединиться: ${SERVER_URL}/auth/signin`,
```

Comment:
serverUrl is a context variable. You don't need to take it from global wariables

Reply:
It is specially left in this form. This variable can be overridden. For a template, this is a common context variable.